src/main/java/edu/ie3/io/factory/EntityFactory.java

```diff
@@ -24,8 +23,9 @@
 
   protected final List<Class<? extends T>> classes;
 
-  public EntityFactory(Class<? extends T>... classes) {
-    this.classes = Arrays.asList(classes);
+  public EntityFactory(Class<? extends T>... allowedClasses) {
+    this.classes = Arrays.asList(allowedClasses);
+    TimeTools.initialize(ZoneId.of("UTC"), Locale.GERMANY, "yyyy-MM-dd HH:mm:ss");
```

Comment:
Do we need this here or does it make more sense in a more universal place?

Reply:
(I realize that might be a lot of work)

Reply:
I _assume_ this is for new (sadly) the only way to go with, as this is the convention introduced for the time formatting for i/o operations. In the long term it might make sense to either a) set this pattern to the standard in utils b) add a second TimeTools pattern that can easily be called c) introduce sth. like "PowerSystemDatamodelContansts" class that hols the pattern as the default one.